packages/platform-ui/src/core/constants.ts

```diff
@@ -4,4 +4,4 @@ export const PRIVACY_POLICY_URL = 'https://per.co.na/pmm/platform-privacy';
 
 export const PASSWORD_MIN_LENGTH = 10;
 
-export const PLATFORM_AUTH_API_BASE_URL = process.env.NODE_ENV === 'development' ? 'https://platform-dev.percona.com' : 'https://platform.percona.com';
+export const PLATFORM_AUTH_API_BASE_URL = process.env.REACT_APP_PLATFORM_AUTH_API_BASE_URL ?? '';
```

Comment:
Actually, we don't need coalescing here.
```suggestion
export const PLATFORM_AUTH_API_BASE_URL = process.env.REACT_APP_PLATFORM_AUTH_API_BASE_URL;
```

Reply:
Why? I think it's needed, since the value can be either string or undefined, so it's either that or the exclamation mark? Otherwise TS will complain about that (and I would prefer solve this on one line, instead of on each line referring to this variable) :)

Reply:
Definitely exclamation mark. What is the reason we want to pass an empty string to the API?